src/ReductionHistory.tsx

```diff
@@ -105,13 +105,21 @@ const ReductionHistory: React.FC = () => {
 
   const fetchReductions = useCallback(async (): Promise<void> => {
```

Comment:
I believe there is at least one more fetch this would be needed on?

Reply:
@keiranjprice101 there is in another branch I'm working on but this branch is off of main.

Reply:
Is there not one in the value editor?